test/versioned/mongodb/bulk.tap.ts

```diff
@@ -0,0 +1,61 @@
+'use strict'
+
+const common = require('./collection-common')
+const mongoPackage = require('mongodb/package.json')
+const semver = require('semver')
+
+common.test('unorderedBulkOp', function unorderedBulkOpTest(t, collection, verify) {
```

Comment:
this is one of the rough edges of our versioned tests but you need to add bulk.tap.js to the files array in `test/versioned/mongodb/package.json`. also these tests need to be semver gated to only run on v3

Reply:
```if(semver.satisfies(mongoPackage.version, '<4')) {```